cernopendata/modules/fixtures/data/records/cms-eventdisplay-files-Run2011A.json

```diff
@@ -0,0 +1,1331 @@
+[
+{
+  "abstract": {
+    "description": "Sample event set from /BTag/Run2011A-12Oct2013-v1/AOD primary dataset in json format readable from the browser-based 3d event display"
+  }, 
+  "accelerator": "CERN-LHC", 
+  "authors": [
+    {
+      "name": "McCauley, Thomas"
+    }
+  ], 
+  "collections": [
+    "CMS-Derived-Datasets"
+  ], 
+  "collision_information": {
```

Comment:
I think this is supposed to be like this
`"collision_information": {
     "energy": "7TeV"
   } `
not 
`"collision_information": {
     "energy": "Collision energy: 7TeV"
   } `

e.g. as in `cms-l1-trigger-information-Run2011A.json`

Reply:
Done.